R/model.R

```diff
@@ -336,6 +347,8 @@ sample_method <- function(data = NULL,
                           adapt_engaged = TRUE,
                           adapt_delta = NULL,
                           metric = NULL,
+                          inv_metric = NULL,
+                          metric_file = NULL,
```

Comment:
Same question as above about the argument order, but can leave as is if we want to promote using `inv_metric` over `metric_file`